txclib/project.py

```diff
@@ -157,14 +165,23 @@ def getset_host_credentials(self, host, username=None, password=None,
             save = True
 
         if save:
+            self._update_file_host(host)
+            self.txrc.set(host, 'username', username)
+            self.txrc.set(host, 'password', password)
+            self.save()
+        return username, password
+
+    def _update_file_host(self, host):
```

Comment:
Shouldn't this method be named `_add_host_to_config_file`. Not the best name, but I think this does not precisely describe what it does...